libsrc/target/zx/rs232/if1/rs232_init.c

```diff
@@ -17,12 +17,16 @@
 u8_t rs232_init()
 {
 #asm
-        rst     8
-        defb    $31             ; Create the IF1 system variables area
-        ;xor     a
-        ;ld      ($5cc7),a       ; Reset SER-FL to clean the input buffer
-        ;ld      a,(23624)
-        ;ld      (23750),a       ; Set IOBORD to the current border colour (hide flashing)
+    PUBLIC BAUD
```

Comment:
This looks like a call to `rs232_init()` will execute `nop ; ld (hl),0 ; nop` which is not really desirable!

To make it ROMable (as well), it can be rearranged:

```
SECTION data_clib

BAUD:  defw $36
SERFL: defw 0

SECTION code_clib
   ld hl,RS_ERR_OK
```

Reply:
Thanks, like I said, new to this. Will update the PR.